docs/release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Release Notes
 
+## Version 15.2.1 (December 3, 2024)
+Multiple bugs were fixed and some recalibration was done since the release of version 15.2.0. Version 15.2.1 will be used for the initial 2025 RP modeling.
```

Comment:
@JoeJimFlood  please change "initial" 2025 RP modeling to "draft" 2025 RP modeling.